kubernetes/policies/general/uses_image_tag_latest.rego

```diff
@@ -24,8 +24,8 @@ __rego_input__ := {
 # have tagged images.
 getTaggedContainers[container] {
 	allContainers := kubernetes.containers[_]
-	[x, y] := split(allContainers.image, ":")
-	y != "latest"
+	tag := split(split(allContainers.image, "@")[0], ":")[1]
```

Comment:
I think we can generally say that if a sha was specified, then we don't care which tag was specified, and this check should pass

Reply:
Agreed.  Updated this section to pass if a sha was specified.  If not, then look for a tag.  Unit tests updated to allow those cases.

Reply:
thanks!